### PR TITLE
Optimize TestApiStatsNetworkStats and TestApiStatsNetworkStatsVersioning

### PR DESCRIPTION
**Pull request by wenchma**
Before Tests results:

```
DockerSuite.TestApiStatsNetworkStatsVersioning                  14.699s
DockerSuite.TestApiStatsNetworkStats                            13.200s
```

After Tests results:

```
PASS: docker_api_stats_test.go:82: DockerSuite.TestApiStatsNetworkStats 6.894s
PASS: docker_api_stats_test.go:128: DockerSuite.TestApiStatsNetworkStatsVersioning  9.521s
```

the results should be accepted.

Related issue #19425 
Signed-off-by: Wen Cheng Ma wenchma@cn.ibm.com

### CODE REVIEW

File: integration-cli/docker_api_stats_test.go [hunk 1]
```diff
@@ -82,14 +82,14 @@ func (s *DockerSuite) TestApiStatsStoppedContainerInGoroutines(c *check.C) {
 func (s *DockerSuite) TestApiStatsNetworkStats(c *check.C) {
 	testRequires(c, SameHostDaemon)
 	testRequires(c, DaemonIsLinux)
-	// Run container for 30 secs
+
 	out, _ := dockerCmd(c, "run", "-d", "busybox", "top")
```
Comment (vdemeester) [inline integration-cli/docker_api_stats_test.go]: @wenchma I think you could even use the new `runSleepingContainer` function for these :angel: 
(https://github.com/docker/docker/blob/0f5aa19593dd2258a26adf37e41e00cbbbab8cf1/integration-cli/docker_utils.go#L1713)

File: integration-cli/docker_api_stats_test.go [hunk 2]
```diff
@@ -128,21 +128,20 @@ func (s *DockerSuite) TestApiStatsNetworkStats(c *check.C) {
 func (s *DockerSuite) TestApiStatsNetworkStatsVersioning(c *check.C) {
 	testRequires(c, SameHostDaemon)
 	testRequires(c, DaemonIsLinux)
-	// Run container for 30 secs
+
 	out, _ := dockerCmd(c, "run", "-d", "busybox", "top")
```
Comment (vdemeester) [inline integration-cli/docker_api_stats_test.go]: Same here

**Comment by vdemeester** [comment]
@wenchma result should be *acceptable* :stuck_out_tongue_closed_eyes: (and thus accepted at some point :wink:)

File: integration-cli/docker_api_stats_test.go (hunk 1 above)
Comment (wenchma) [inline integration-cli/docker_api_stats_test.go]: @vdemeester , will cover it

**Comment by thaJeztah** [comment]
ping @vdemeester I see this PR was updated, PTAL

**Comment by wenchma** [comment]
just have a rebase and retrriger the jenkins

**Comment by wenchma** [comment]
@thaJeztah can you retrigger the `windows TP4` again ? :stuck_out_tongue:

**Comment by vdemeester** [comment]
SGTM, ping @runcom @calavera

**Comment by calavera** [comment]
LGTM

**Comment by LK4D4** [comment]
LGTM

**Pull request merged**